.github/workflows/tests.yml

```diff
@@ -18,17 +18,8 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ["3.9", "3.10"]
```

Comment:
We should keep Python 3.8 in the array of supported/tested versions, as it's still supported and common, and there's no real reason to remove it, as far as I can see.